modules/dcache/src/main/java/org/dcache/pool/statistics/IoStatistics.java

```diff
@@ -0,0 +1,205 @@
+package org.dcache.pool.statistics;
+
+import java.util.ArrayList;
+import java.util.Collections;
+
+/**
+ * This class stores statistics about read an write processes
+ */
+
+public class IoStatistics {
+
+    private long _readRequestNum = 0;
+    private long _readBytes;
+    private ArrayList<Float> _readSpeeds = new ArrayList<Float>();
```

Comment:
Hi, no there is no reason for using float so we will change all of them to double. Thanks for the hint :)